Documentation/ApiOverview/SiteHandling/AccessingSiteConfiguration.rst

```diff
@@ -1,108 +1,121 @@
 
-   // current site
-   $site = $request->getAttribute('site');
+    // current site
+    $site = $request->getAttribute('site');
 
-   // current site language
-   $siteLanguage = $request->getAttribute('language');
+    // current site language
+    $siteLanguage = $request->getAttribute('language');
 
 
 ..  versionchanged:: 11.3
-    Since TYPO3 v11.3 the :ref:`Extbase <extbase>` request class implements the
+    The :ref:`Extbase <extbase>` request class implements the
     PSR-7 :php:`\Psr\Http\Message\ServerRequestInterface`. Therefore you can
     retrieve all needed attributes from the request object.
 
+
 .. index:: pair: Site handling; SiteFinder
 
 Finding a site object
 =====================
 
-When you need to access site configuration for a specific page ID or by identifier,
-you can use the :php:`SiteFinder` (:php:`\TYPO3\CMS\Core\Site\SiteFinder`).
+When you need to access the site configuration for a specific page ID or by an
+identifier, you can use the :php:`\TYPO3\CMS\Core\Site\SiteFinder` class.
 
-The :php:`SiteFinder` offers the following methods for finding a site:
+The site finder offers the following methods for finding a site:
 
-- :php:`getSiteByIdentifier()`: returns site object for the specified identifier ("folder name")
-- :php:`getSiteByRootPageId()`: returns site object for a specific root page (pid = 0 or is_siteroot set)
-- :php:`getSiteByPageId()`: returns site object for a page (walks the root line to find the root page
-  and returns the site configuration)
-- :php:`getAllSites()`: returns all configured site objects
+-   :php:`getSiteByIdentifier()`: returns the site object for the specified
+    identifier ("folder name")
+-   :php:`getSiteByRootPageId()`: returns the site object for a specific root
+    page (`pid = 0` or `is_siteroot` set)
+-   :php:`getSiteByPageId()`: returns the site object for a page (walks the root
+    line to find the root page and returns the site configuration)
+-   :php:`getAllSites()`: returns all configured site objects
 
-All methods for finding a specific site throw an exception if no site was found.
+All methods for finding a specific site throw an exception, if no site was found.
 
 
 .. index:: pair: Site handling; Site object
 
-The Site object
+The site object
 ===============
 
 Now we know how to find a :php:`Site` object, but what can it do?
 
 First of all, it gives us access to the site configuration options via
 
-- :php:`getConfiguration()`: returns the complete configuration
-- :php:`getAttribute()`: returns a specific configuration attribute (root level configuration only)
+-   :php:`getConfiguration()`: returns the complete configuration
+-   :php:`getAttribute()`: returns a specific configuration attribute (root
+    level configuration only)
 
-It additionally provides methods for accessing related objects (languages / errorHandling):
+Additionally, it provides methods for accessing related objects (languages /
```

Comment:
```suggestion
Additionally, the :php:`Site` provides methods for accessing related objects (languages /
```
I would repeat this here, it was two paragraphs up we mentioned what we talk about